bitnami/wavefront/templates/collector-cluster-role.yaml

```diff
@@ -38,7 +38,22 @@ rules:
       - create
       - list
       - watch
-
+{{- if .Values.podSecurityPolicy.enabled }}
```

Comment:
The value is `.Values.podSecurityPolicy.create`. This should be changed, right? Same thing in `bitnami/wavefront/templates/podsecuritypolicy.yaml`

Reply:
good catch!